### PR TITLE
📖 Add a README for contributor-facing documentation on adding strings/translations for stories

### PR DESCRIPTION
**Pull request by newmuis**

### CODE REVIEW

File: extensions/amp-story/1.0/_locales/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+* `en-GB` (English, Great Britain)
+* `en` (English)
+* `default` (Default)
+
+The `default` language code is a fallback used in the case that the publisher-specified language code does not exist.  It uses a minimal amount of English strings, so that the document can be displayed mostly in its primary language.  Any labels that describe otherwise familiar or intelligible iconography can be dropped entirely.  For example, since sharing icons show the sharing network's logo (e.g. the Twitter logo), the "Twitter" string is redundant and can be left out of the `default` language.
+
+# Viewing the current strings
+
+You can view the translations provided for each language in [this spreadsheet](https://bit.ly/amp-story-strings).  Any cells with the text `undefined` mean that the string will not be shown in the specified language, and any fall
```
Comment (CrystalFaith) [inline extensions/amp-story/1.0/_locales/README.md]: Looks like the last sentence was cut off?

File: extensions/amp-story/1.0/_locales/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+/* ... */
+```
+
+4. Send a pull request with your changes
+
+
+# Adding new translations (non-English strings)
+
+1. Find which string(s) are missing by looking at [the string spreadsheet](https://bit.ly/amp-story-strings)
+2. Open the `*.js` file from the [`_locales` directory](https://github.com/ampproject/amphtml/tree/master/extensions/amp-story/1.0/_locales) for the language you would like to add a translation for
```
Comment (CrystalFaith) [inline extensions/amp-story/1.0/_locales/README.md]: was this one cut off too?

File: extensions/amp-story/1.0/_locales/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+/* ... */
+
+[LocalizedStringId.MY_LOCALIZED_STRING_ID]: {
+  string: 'My string',
+},
+
+/* ... */
+```
+
+4. Send a pull request with your changes
```
Comment (CrystalFaith) [inline extensions/amp-story/1.0/_locales/README.md]: These last two sections have inconsistent punctuation for the numbered bullets, could we end them each with `.`?

File: extensions/amp-story/1.0/_locales/README.md (hunk 2 above)
Comment (newmuis) [reply extensions/amp-story/1.0/_locales/README.md]: Not cut off, just bad grammar 😅 

Fixed it up a little

File: extensions/amp-story/1.0/_locales/README.md (hunk 1 above)
Comment (newmuis) [reply extensions/amp-story/1.0/_locales/README.md]: Whoops!  Fixed.

**Pull request merged**